pkg/oc/cli/cmd/rsync/util.go

```diff
@@ -84,14 +84,26 @@ func rsyncFlagsFromOptions(o *RsyncOptions) []string {
 	return flags
 }
 
-func rsyncSpecificFlags(o *RsyncOptions) []string {
+func tarFlagsFromOptions(o *RsyncOptions) []string {
 	flags := []string{}
+	if !o.Quiet {
+		flags = append(flags, "-v")
+	}
 	if len(o.RsyncInclude) > 0 {
-		flags = append(flags, "--include")
+		for _, include := range o.RsyncInclude {
+			flags = append(flags, fmt.Sprintf("**/%s", include))
```

Comment:
does this make the behavior consistent with rsync, or with our doc?

Reply:
@smarterclayton 
Will update to match rsync behavior.

From local testing, the current include / exclude rsync behavior is as follows:

```
$ oc rsync --include="*.yaml" --exclude="ignore.*" mypod-1-x29xk:/tmp/myplugin ./
receiving incremental file list
myplugin/
myplugin/LICENSE
myplugin/ignore.yaml
myplugin/plugin.yaml
```

Wanted to double-check if this was behaving as we expect? (I would have expected `myplugin/ignore.yaml` to not have been included)